x/concentrated-liquidity/lp_test.go

```diff
@@ -1152,9 +1151,9 @@ func (s *KeeperTestSuite) TestSendCoinsBetweenPoolAndUser() {
 			// store pool interface
 			poolI, err := s.App.ConcentratedLiquidityKeeper.GetPoolById(s.Ctx, 1)
 			s.Require().NoError(err)
-			concentratedPool, ok := poolI.(types.ConcentratedPoolExtension)
+			concentratedPool, ok := poolI.(types.ConcentratedPoolExtension) //nolint:gosimple // need to check if poolI is a ConcentratedPoolExtension
```

Comment:
out of curiosity, why do we use gosimple here?

Reply:
Okay very good question. This one was especially challenging. I'm going to need to answer with about a 50%. I don't know, but you know the "ok ,"pattern?

I know that it has something to do with that, and apparently this is like not the world's most necessary type assertion because it already has that type, but when I basically remove the type assertion, I would get fireworks. Do you have any ideas on how to fix it?

Reply:
Sorry for the garbled reply before -- the exact reason is that it is already of that type, so the type assertion throws off the linter.

Reply:
I'm resolving this, but please let me know if you'd like the test itself changed.